django/db/backends/oracle/operations.py

```diff
@@ -575,6 +575,8 @@ def combine_expression(self, connector, sub_expressions):
             return 'FLOOR(%(lhs)s / POWER(2, %(rhs)s))' % {'lhs': lhs, 'rhs': rhs}
         elif connector == '^':
             return 'POWER(%s)' % ','.join(sub_expressions)
+        elif connector == '#':
+            return 'BITXOR(%s)' % ','.join(sub_expressions)
```

Comment:
`BITXOR` doesn't exist on Oracle (see [docs](https://docs.oracle.com/en/database/oracle/oracle-database/18/sqlrf/Single-Row-Functions.html#GUID-B93F789D-B486-49FF-B0CD-0C6181C5D85C)). I think we can emulate it with `BITAND()`, like we do for `|`.

Reply:
Ah, I must've misread the documentation somehow. I guess it could be implemented as:
```python
(%(lhs)s  + %(rhs)s) - BITAND(%(lhs)s , %(rhs)s) * 2
```

But the problem I'm seeing is that the right-hand side parameter needs to be repeated when passed to `cx_Oracle`.

```python
execute('SELECT ... ("table.field"  + %s) - BITAND("table.field" , %s) * 2', [42])
```

Reply:
That shouldn't be an issue (see a workaround for `|`).

Reply:
Maybe I'm just tired but I fail to see how XOR can be simplified to only need one reference to the right-hand side parameter.

Reply:
Ahh right :disappointed: I'm fine with raising exception on Oracle:
```python
raise NotSupportedError('Bit-wise xor is not supported in Oracle.')
```

Reply:
I added the same exception for SQLite. I also updated the documentation to point out these limitations.